cdap-docs/admin-manual/source/_includes/installation/ha-installation.txt

```diff
@@ -0,0 +1,41 @@
+Enabling CDAP HA
+----------------
+In addition to having a :ref:`cluster architecture <admin-manual-install-deployment-architectures-ha>`
+that supports HA (high availability), these additional configuration steps need to followed and completed:
+
+CDAP Master
+...........
+- Install multiple instances of the CDAP Master on different nodes.
```

Comment:
For each section, I would suggest separating out the general information as an intro, followed by the steps (with potentially more detail).

CDAP Master
CDAP Master primarily performs coordination tasks and can be scaled for redundancy.  The instances coordinate amongst themselves, electing one as a leader at all times.
- Install the cdap-master package on different nodes
- Ensure they are configured identically (/etc/cdap/conf/cdap-site.xml).  No additional configuration is required for HA.
- Start the cdap-master service on each node